doc/specifications/proposed/named_queries/named_queries.md

```diff
@@ -12,6 +12,15 @@ ContentType identifier...
 In the initial version, QueryType objects can be created as PHP classes. The main use-case is to allow developers
 to define Queries for the project that can be used in several ways, without knowing their internals.
 
+### Differnces from Persisted Queries
+
+When `Views` where added to REST API there was the intention to allow these to be persisted at some point (after UI where moved to Platform Stack). And while there are no own specification for it yet, these are the main differences between the two:
+- Persisted Queries serves as Editorial Queries/Search for future possible features like reuse in content, blocks, and plainly sharing link to a view to co workers or public.
+- Named Queries on the other side is a DX feature for devloper provided queries for reuse in code, and aim to make eZ Platfrom much easier to work with by exposing the power of `filtering` views direclty from templates.  
+- Persisted Queries, unlike Named Queries, will by it's nature be possible to be retrive and created from PHP Repository API, and hence also REST API which will need distingusing these two from eachother for UI's exposing such  editing functionaity.
+- Persisted Queries has the possibility to allow Repository to keep track of hits internally in the future, ala how JIRA works with filters. This will allow more advance cache clearing logic to aim for editor never having to take care about cache.
```

Comment:
I'm frankly a bit lost now. See these as open questions for tomorrow.

> I also think persisted filters is more for specific things like latest blogs, while named/alias is for generic once like children.

Doesn't this more depend on who's creating the Query, and how it is used ? In most cases, I'd think that a developer would use the file based approach, and an editor would use the persisted one.

As a developer, if I need a list of the latest blogs, I'd expect that I can create a query for it without persisting it in the database. I'd then use this query in a couple places, where I need them, mostly using sub-queries.

As an editor, maybe I'd like to list the latest blog posts in some block that I add to a RichText. 

Am I getting this right ? One of the overlaps we have discussed in REST's `content/views`. I defended keeping it as a unique entry points for all content queries, but it might make sense that we actually separate named queries here. After all, they're not real "views" as described in this document, they're just searches.

But then, does it make sense to allow the controller (see specs) to execute both named queries and views ? From a developer's standpoint, if you can't rely on the query's contents/behavior, you aren't able to use it from templates (you've raised this before).